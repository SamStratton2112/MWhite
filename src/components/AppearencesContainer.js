import ListModal from './ListModal';
import mike from '../data';
import {Container} from 'react-bootstrap';

const AppearencesContainer = ()=>{
    console.log(mike)
    return(
        <Container>
            <span>Appearences...</span>
            <br/>
            <ListModal name={mike.dj.appearences.name} list={mike.dj.appearences.list}>{mike.dj.appearences.name}</ListModal>
            <ListModal name={mike.dj.openedFor.name} list={mike.dj.openedFor.list}>{mike.dj.openedFor.name}</ListModal>
            <ListModal name={mike.dj.djFor.name} list={mike.dj.djFor.list}>{mike.dj.djFor.name}</ListModal>
        </Container>
    )

};

export default AppearencesContainer;