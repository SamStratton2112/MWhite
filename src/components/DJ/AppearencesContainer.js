import ListModal from '../ListModal';
import mike from '../../data';
import {Container} from 'react-bootstrap';

const AppearencesContainer = ()=>{
    return(
        <Container>
            <h3 style={{marginTop: '1em'}}>Appearences...</h3>
            <ListModal name={mike.dj.appearences.name} list={mike.dj.appearences.list}>{mike.dj.appearences.name}</ListModal>
            <ListModal name={mike.dj.openedFor.name} list={mike.dj.openedFor.list}>{mike.dj.openedFor.name}</ListModal>
            <ListModal name={mike.dj.djFor.name} list={mike.dj.djFor.list}>{mike.dj.djFor.name}</ListModal>
        </Container>
    )
};

export default AppearencesContainer;