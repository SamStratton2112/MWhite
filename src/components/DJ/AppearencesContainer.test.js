import AppearencesContainer from './AppearencesContainer';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(AppearencesContainer, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <AppearencesContainer />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <AppearencesContainer />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});