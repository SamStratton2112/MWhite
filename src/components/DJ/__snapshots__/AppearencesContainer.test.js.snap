// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppearencesContainer matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <button
      class="bttn"
    >
      As a Guest D.J.
    </button>
    <button
      class="bttn"
    >
      As an Opening D.J.
    </button>
    <button
      class="bttn"
    >
      As a Featured D.J.
    </button>
  </div>
</DocumentFragment>
`;
